app/services/values/space.rb

```diff
@@ -7,7 +7,7 @@ def initialize(coordinates)
     @status      = "Not Attacked"
   end
 
-  def attack!
+  def atack!
```

Comment:
🤔

Reply:
Huh. Whoops. I'll mark that to get it fixed